src/components/form-elements/_select.scss

```diff
@@ -1,7 +1,7 @@
 $selectHeight: rhythm(3 / 2);
 $selectBackground: $white;
 $selectTextColor: $grayPrimary;
-$selectFontSize: fontSize(small);
```

Comment:
`fontSize(default)` can also work but I want to be more explicit for the future.